scss/elements/tables.scss

```diff
@@ -0,0 +1,23 @@
+.table {
+  background-color: #fff;
+
+  &.is-centered th {
+    text-align: center;
+  }
+  &.is-bordered {
+    box-shadow: 4px 0 #212529, 0 -4px #212529, -4px 0 #212529, 0 4px #212529;
+
+    th,
+    td {
+      padding: 8.5px;
+    }
+
+    tr th {
+      box-shadow: 0 0 #212529, 0 0 #212529, -4px 0 #212529, 0 0 #212529;
+    }
+
+    tr td {
+      box-shadow: 0 0 #212529, 0 -4px #212529, 0 0 #212529, -4px 0 #212529;
+    }
```

Comment:
I fix to prevent lines from overlapping and deleted unnecessary values. 🔧 
```
    tr {
      th + th {
        box-shadow: -4px 0 $base-color;
      }
      td {
        box-shadow: 0 -4px $base-color;
      }
      td:not(:first-child) {
        box-shadow: -4px 0 $base-color, 0 -4px $base-color;
      }
    }
```